api/ReadableByteStreamController.json

```diff
@@ -11,6 +11,10 @@
           "chrome_android": {
             "version_added": "89"
           },
+          "deno": {
+            "version_added": "1.0",
+            "notes": "This interface is not exposed on the global scope on versions prior to 1.11."
```

Comment:
If instances are available, but `ReadableByteStreamController` isn't exposed, then I'd use two support statements:

1. Supported from version 1.11
2. version_added 1.0; version_removed 1.11 with `partial_implementation` and this note (without mentioning the version number)